doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -629,12 +629,24 @@ file is a particular case of a module called a *library file*.
 
    Loads an OCaml plugin and its dependencies dynamically.  The :n:`@string`
    argument must be a valid `findlib <http://projects.camlcity.org/projects/findlib.html>`_
-   plugin name, for example ``coq-core.plugins.ltac``. The first component of
-   the plugin name is a package that has to be in scope of ``findlib``'s'
-   search path. As of Coq 8.16, the command also support a "legacy"
+   plugin name, for example ``coq-core.plugins.ltac``. As of Coq 8.16,
+   the command also support a "legacy"
    syntax compatible with the plugin loading system used in Coq
    8.0-8.15, see below.
 
+   The first component of the plugin name is a package that has to be in
+   scope of ``findlib``'s' search path. Conventionally, it is named
+   ``coq-pkg`` where ``pkg`` is the name of the plugin. It must
+   correspond to the name of the ``META.coq-pkg`` file.
+
+   The other components correspond to the name given to the package in
+   the ``META.coq-pkg`` file. By default, if the ``META`` file is
+   generated by the ``-generate-meta-for-package`` option of
+   ``coq_makefile``, this name is ``plugin``.
+
+   The full path must be given as the plugin name in the ``.mlg`` file
+   of the plugin: ``DECLARE PLUGIN "coq-pkg.plugin"``.
```

Comment:
I'm not sure this is the best place for this, but unfortunately we don't have documentation for coqpp other than the plugin tutorial so I guess it is fine.